engine/build.gradle

```diff
@@ -136,6 +136,7 @@ dependencies {
     compile group: 'org.terasology', name: 'gestalt-module', version: '5.1.3'
     compile group: 'org.terasology', name: 'gestalt-asset-core', version: '5.1.3'
     compile group: 'org.terasology', name: 'TeraMath', version: '1.4.0'
+    compile 'org.joml:joml:1.9.11'
```

Comment:
I recommend using the latest JOML 1.9.19. There have been a whole host of performance improvements and also bug fixes between 1.9.11 and 1.9.19. See the release notes of the releases under https://github.com/JOML-CI/JOML/releases for an exhaustive list of all the changes between those two versions.